### ISSUE TITLE
GUI: Error dialog when loading NFT with unresolvable DNS

### ISSUE THREAD

**Issue by paninaro** [Chia-Network/chia-blockchain-gui#1060]
Failure to load metadata when the URL's host is unresolvable leads to the following modal error dialog:

![Image](https://user-images.githubusercontent.com/339312/196255043-0943398b-9f43-4377-81b8-988e14eed2fc.png)

We should be communicating the error state in the NFTCard/NFTPreview

**Comment by paninaro** [comment]
NFT details:

```
NFT identifier:            nft1rdxsmqg8nxu52pmftg9lu4zm2xregmjav7rxrwng0fucs2seuh0qc5lak9
Launcher coin ID:          1b4d0d810799b94507695a0bfe545b5187946e5d678661ba687a79882a19e5de
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       224146a63d464ddd50f1945ae76e8e6cc057c85bf6c467aed87f98ba5513287b
On-chain data/info:        ((117 "https://bafybeialb6lxafyacjc7unyoahnzlzsjjp47u4vu37dj4xks2vsif2f6uu.ipfs.nftstorage.link/3155.png") (104 . 0x600dcfbbc55ea670b4a615b0fdc851a60a479215b2327eccbdefa3a12a081c62) (28021 "http://garza.biz/") (27765) (29550 . 1) (29556 . 1) (28008 . 0x15748b116a4ba17b5d2fdae8f5f91be2346e8b7d5e6cf9544ffab244fe9db43b))
Owner DID:                 None
Minter DID:                None
Royalty percentage:        0
Royalty puzhash:           a888095d8ca6589f0e531d515b0594f5893259042574aaf2db48e5ceb577b2c1
NFT content hash:          600dcfbbc55ea670b4a615b0fdc851a60a479215b2327eccbdefa3a12a081c62
Metadata hash:             15748b116a4ba17b5d2fdae8f5f91be2346e8b7d5e6cf9544ffab244fe9db43b
License hash:
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2612733
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://bafybeialb6lxafyacjc7unyoahnzlzsjjp47u4vu37dj4xks2vsif2f6uu.ipfs.nftstorage.link/3155.png

Metadata URIs:
   http://garza.biz/

License URIs:
```

**Comment by paninaro** [comment]
Screen recording showing the error dialog:

https://user-images.githubusercontent.com/339312/197656927-0d2397f2-ac7c-4b8d-9546-3441018173d5.mov

**Comment by emlowe** [comment]
Fixed in 1.6.1
